appengine/go11x/tasks/create_http_task/create_http_task.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START cloud_tasks_create_http_task]
+
+// Command create_task constructs and adds a task to an App Engine Queue.
+package main
+
+import (
+	"context"
+	"fmt"
+
+	cloudtasks "cloud.google.com/go/cloudtasks/apiv2beta3"
+	taskspb "google.golang.org/genproto/googleapis/cloud/tasks/v2beta3"
```

Comment:
Still using beta3 API.

Reply:
That's the correct version for this feature.